test/common/common/base64_test.cc

```diff
@@ -74,6 +75,16 @@ TEST(Base64Test, Decode) {
   }
 }
 
+TEST(Base64Test, DecodeFailure) {
+  EXPECT_EQ("", Base64::decode("==Zg"));
+  EXPECT_EQ("", Base64::decode("=Zm8"));
+  EXPECT_EQ("", Base64::decode("Zm=8"));
+  EXPECT_EQ("", Base64::decode("Zg=A"));
+  EXPECT_EQ("", Base64::decode("Zm9=")); // 011001 100110 111101 <- unused bit at tail
```

Comment:
Does this one test case cover both of the unused bit at tail cases above? Seems like there should be two but maybe I'm missing something.